vllm/engine/async_llm_engine.py

```diff
@@ -540,11 +540,13 @@ async def engine_step(self, virtual_engine: int) -> bool:
             request_outputs = await self.engine.step_async(virtual_engine)
 
         # Put the outputs into the corresponding streams.
+        finished = False
         for request_output in request_outputs:
             self._request_tracker.process_request_output(
                 request_output, verbose=self.log_requests)
+            finished = finished or request_output.finished
```

Comment:
should this be and, actually? we should stop only if all requests are finished.

Reply:
Yes. I have rectify it, Thanks.